mmpose/datasets/datasets/__init__.py

```diff
@@ -1,5 +1,6 @@
 from ...deprecated import (TopDownFreiHandDataset, TopDownOneHand10KDataset,
                            TopDownPanopticDataset)
+from .body3d import Body3DH36MDataset
```

Comment:
Body or Pose? What do you think @jin-s13

Reply:
Body is better.